docs/linux/BUILDING.md

```diff
@@ -0,0 +1,46 @@
+# CentOS 7:
+
+CentOS 7 ships with older versions of glibc, cmake and gcc that make it difficult to compile this project and its dependencies. Building on CentOS involves providing newer packages built by the upstream projects, or though an available Software Collection Library(SCL).
+
+## Example: Building the `ziti-edge-tunnel` binary:
+
+
+```bash
+#We'll install CMake to this prefix, so let's make sure it's in the PATH
+export PATH="/opt/cmake:${PATH}"
```

Comment:
a variable for "/opt/cmake" wouldn't be a bad idea

Reply:
Done.